src/datatypes/map.c

```diff
@@ -255,24 +234,30 @@ XXH64_hash_t Map_HashCode
 	return hashCode;
 }
 
+// Utility function to increase the size of a buffer.
+static inline void _Map_ExtendBuffer(
```

Comment:
Please rename to `static inline void _ExtendBuffer` as this has nothing to do with the `Map` object